packages/react-tinacms-github/README.md

```diff
@@ -227,4 +258,99 @@ const formOptions = {
      },
      //...
 }
+```
+
+## _GithubFile_ and _useGithubFile_
+
+The `GithubFile` class is a helper class to provide methods for manipulating a single file in your GitHub repo.
+
+### Signature
+
+The `GithubFile` constructor takes four parameters: `cms`, `path`, `parse`, and `serialize`.
+
+| argument | description |
+| --- | --- |
+| cms | An instance of TinaCMS |
+| path | Filepath, relative to repository root |
+| parse | Function to deserialize file contents into an object |
+| serialize | Function to serialize data object into a string |
+
+### Methods
+
+The `GithubFile` class has two public methods:
+
+| method | description |
+| --- | --- |
+| fetchFile | Wraps `GithubClient#fetchFile`; async function to retrieve file contents from GitHub API |
+| commit | Wraps `GithubClient#commit`; async function to commit changes to a file |
+
+
+
+### Usage Example
+
+```ts
+import { GithubFile } from 'react-tinacms-github'
+
+async function example() {
+  const navigationFile = new GithubFile(
+    cms,
+    'content/navigation.json',
+    JSON.parse,
+    JSON.stringify
+  )
+
+  // get file contents from GitHub
+  const navigation = await navigationFile.fetchFile()
```

Comment:
This confused me when I read it, which makes me think that maybe the API should be `navigationFile.fetchData` or `navigationFile.fetchContent`

Reply:
Yeah, I feel the same, but I thought it better to make the naming consistent w/ the GithubClient method. Maybe not though.